app/design/frontend/Diana/default-theme/Magento_Cms/templates/store-selector-main.phtml

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/** @var \Magento\Framework\View\Element\Template $block */
+?>
+<div data-bind="scope: 'renderStores'" class="render-stores-wrapper">
+    <h2 class="title-store-selector">
+        <?= $block->escapeHtml(__('Sales are available at stores in ')) ?>
+        <span data-bind="text: selectedCity"></span>
+    </h2>
+    <div class="grid-of-stores" data-bind="foreach: paginatedElements">
+        <div class="available-store">
+            <a data-bind="attr: {href: link}">
+                <img class="image-store" data-bind="attr: {alt: name}" width="180px"
+                     src="<?= $block->getViewFileUrl('images/example-store.jpeg'); ?>" />
+            </a>
+            <div class="description-of-store">
+                <strong data-bind="text: name"></strong>
+<!--                <br>-->
+                <span data-bind="text: mall"></span>
+<!--                <br>-->
+                <address data-bind="text: address"></address>
+                <br>
+                <time data-bind="text: schedule"></time>
+<!--                <br>-->
+                <a data-bind="text: phone, attr: {href: 'tel:' + phone}"></a><br>
+                <a class="more-details" href="attr: {href: link}"><?= $block->escapeHtml(__('More details')) ?>&#160;&#10230;</a>
+            </div>
+        </div>
+    </div>
+    <div class="pagination-wrapper">
+        <a class="pagination-arrow link-previous-page" data-bind="click: previousPage">&lt;</a>
+        <span data-bind="foreach: quantityOfPagesRenderer">
+            <span class="page-number" data-bind="text: numberOfPage, click: $parent.linkToPageNumber.bind($parent), style: {'font-weight': isCurrent() ? '900': '400'}">
```

Comment:
@Dianaeva To avoid inline styling, please use CSS class with appropriate styling